egov-persister/notification-persister.yml

```
@@ -0,0 +1,81 @@
+
+        - query: INSERT INTO dristi_notification_document(id,fileStore,documentUid,documentType ,notification_id,additionalDetails) VALUES (?,?, ?,?,?,?) ON CONFLICT (id) DO UPDATE SET fileStore = EXCLUDED.fileStore,documentUid = EXCLUDED.documentUid,documentType = EXCLUDED.documentType,additionalDetails = EXCLUDED.additionalDetails;;
+          basePath: $.notification.documents.*
+          jsonMaps:
+            - jsonPath: $.notification.documents.*.id
+            - jsonPath: $.notification.documents.*.fileStore
+            - jsonPath: $.notification.documents.*.documentUid
+            - jsonPath: $.notification.documents.*.documentType
+            - jsonPath: $.notification[?({id} in @.documents[*].id)].id
+            - jsonPath: $.notification.documents.*.additionalDetails
+              type: JSON
+              dbType: JSONB
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)